Blammo/package.yaml

```diff
@@ -1,7 +1,7 @@
 name: Blammo
-version: 1.2.1.0
+version: 2.0.0.0
```

Comment:
Just to make sure we're on the same page: our [versioning scheme](https://illuminate.atlassian.net/wiki/spaces/EN/pages/17185952648/Maintaining+Haskell+Open+Source#Versioning) is:

```
{EPOCH}.{MAJOR}.{MINOR}.{PATCH}
```

We follow all the SemVer rules in the major/minor/patch parts and that ends up satisfying PVP simply because `{EPOCH}.` exists, regardless of what it is or when it's bumped. So we only bump it when there's a "large rewrite" or some "marketing" reason to do so. This makes most of our packages forever `0.` or `1.` prefixed.

So I guess I just wanted to make sure you know you _could_ technically ship this as `1.3` and are intentionally choosing the bigger bump because you feel the package break up is in fact significant enough.

Reply:
Yes, that is my thinking. Since this will require packaging changes in anything that uses the moved module, it seems like a particularly noteworthy change. I think my threshold for an epoch is lower than most; this version component is generally underutilized IMO.